src/main/java/org/broadinstitute/hellbender/tools/copynumber/formats/records/CalledLegacySegment.java

```diff
@@ -0,0 +1,46 @@
+package org.broadinstitute.hellbender.tools.copynumber.formats.records;
+
+import org.broadinstitute.hellbender.utils.SimpleInterval;
+
+public class CalledLegacySegment extends LegacySegment {
+
+    private CalledCopyRatioSegment.Call call;
+
+    public CalledLegacySegment(final String sampleName, final SimpleInterval interval, final int numProbes, final double segmentMean,
```

Comment:
Can you add white space to align these parameters?

Reply:
Done